xmipp

```diff
@@ -761,10 +761,11 @@ def configCuda(configDict):
             if checkProgram("nvcc"):
                 nvccVersion, nvccFullVersion = getCudaVersion("nvcc")
                 print(green('CUDA-' + nvccFullVersion + ' detected.'))
-                if nvccVersion != 8.0 and not 'XMIPP_ALLOW_ANY_CUDA' in os.environ:
+                if nvccVersion != 8.0 and not os.environ.get('XMIPP_ALLOW_ANY_CUDA', 'False') == 'True':
```

Comment:
we should get rid of code / restrictions like this ...

Reply:
For me, we should let compiling with any cuda by default (printing a warning if we detect some possible incompatibility).

However, if we check the cuda-version in order to prevent fails in the compilation time, I think that we should let try with any version without warranties in some easy way. Sometimes I have installing / supporting scipion/xmipp in a facility and they want not change cuda version and I had to manipulate the code in order to bypass the check and this is not comfortable.